model_analyzer/analyzer.py

```diff
@@ -64,7 +70,8 @@ def __init__(self, config, server, state_manager, checkpoint_required):
         self._result_manager = ResultManager(config=config,
                                              state_manager=self._state_manager)
 
-    def profile(self, client, gpus):
+    def profile(self, client: Union[TritonGRPCClient, TritonHTTPClient],
```

Comment:
For the case of multiple implementations of a class, we should just type-hint the parent/interface. In this case, that is just TritonClient.

Reply:
Done